src/pages/ResetPasswordPage/ResetPasswordPage.jsx

```diff
@@ -26,7 +26,7 @@ export const ResetPasswrodPage = () => {
       Swal.fire('Error en registro', 'Contraseñas no son iguales', 'error');
```

Comment:
Please review this suggested change. Without context, I think "Error de autenticación" sounds better, but it is all up to you.

```suggestion
      Swal.fire('Error de autenticación', 'Las contraseñas no son iguales', 'error');
```